configs/apache_spark.json

```diff
@@ -4,7 +4,7 @@
     {
       "url": "https://spark.apache.org/docs/(?P<version>.*?)/",
       "variables": {
-        "version": ["latest", "3.0.3",  "3.1.1", "3.1.2", "3.2.0"]
+        "version": ["latest", "3.0.3", "3.1.1", "3.1.2", "3.2.0", "3.2.1"]
```

Comment:
I think we remove 3.2.0(?)

Reply:
@HyukjinKwon I don't understand. We should keep it.

Reply:
Oops. sorry I misunderstood. LGTM.